dojo/settings/settings.dist.py

```diff
@@ -883,7 +885,7 @@ def saml2_attrib_map_format(dict):
         SAML_METADATA['local'] = [env('DD_SAML2_METADATA_LOCAL_FILE_PATH')]
     INSTALLED_APPS += ('djangosaml2',)
     MIDDLEWARE.append('djangosaml2.middleware.SamlSessionMiddleware')
-    AUTHENTICATION_BACKENDS += ('djangosaml2.backends.Saml2Backend',)
+    AUTHENTICATION_BACKENDS += env('DD_SAML2_AUTHENTICATION_BACKENDS',)
```

Comment:
Looks like you are missing a parenthesis 
```suggestion
    AUTHENTICATION_BACKENDS += (env('DD_SAML2_AUTHENTICATION_BACKENDS'),)
```

Reply:
Hi @Maffooch
Yes, that was my mistake. Added your suggestions
Thank you!